src/Compiler/CompilerUtils.cpp

```diff
@@ -251,6 +237,24 @@ struct Command {
 // =============================================================================
 // Methods for compiling and file processing.
 
+std::string getOnnxMlirFullVersion(bool simplify) {
+  const std::string OnnxMlirVersion = "onnx-mlir version 1.0.0";
+#ifdef ONNX_MLIR_VENDOR
+  return ONNX_MLIR_VENDOR ", " + OnnxMlirVersion;
+#elif defined(ONNX_MLIR_REPOSITORY) && defined(ONNX_MLIR_REVISION) &&          \
+    defined(LLVM_REPOSITORY) && defined(LLVM_REVISION)
+  if (simplify) {
```

Comment:
Thinking to either
1. create another cmake variable so vendor can decide to always use the simplified version instead of a boolean for only this one callsite. 
2. always use the simplified version, but that become inconsistent with how llvm does it. 

Please let me know how you think.